src/ng/templateRequest.js

```diff
@@ -86,7 +88,7 @@ function $TemplateRequestProvider() {
           cache: $templateCache,
           transformResponse: transformResponse
         }, httpOptions))
-        ['finally'](function() {
+        .finally(function() {
```

Comment:
Yeah, most/all of the them where introdiced as part of the Possibly Unhandled Rejection feature.
I don't think we have any in `v1.5.x` though. So, since we intent to backport this to 1.5.x, I wouldn't change this.